.github/workflows/systemd-verify.yaml

```diff
@@ -0,0 +1,22 @@
+name: Verify systemd files
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
```

Comment:
We won't benefit from running this job if no systemd files changed, will we? Considering skipping it in those cases: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore.